tests/e2e/local/vagrant/.gitignore

```diff
@@ -0,0 +1 @@
+.vagrant
```

Comment:
What does it mean?

Reply:
the common use case of someone `cd`ing into `test/e2e/local/vagrant` and then executing `sh setup_vm.sh` will create a `.vagrant` directory in the pwd. This dir is never something you'd want to commit, so it's better to gitignore it to prevent mistakes and provide better experience to e2e testers.

Reply:
Please add new line after this entry